cmd/terramate/cli/cli.go

```diff
@@ -676,6 +660,22 @@ func (c *cli) gitSafeguards(checks terramate.RepoChecks, shouldAbort bool) {
 	}
 }
 
+func (c *cli) gitRemoteSafeguard() {
+	logger := log.With().
+		Str("action", "checkGit()").
+		Logger()
+
+	if !c.prj.isRepo || !c.gitSafeguardRemoteEnabled() {
+		return
+	}
+
+	logger.Trace().Msg("check git default branch is updated")
+
+	if err := c.prj.checkLocalDefaultIsUpdated(); err != nil {
```

Comment:
the naming of the functions is very confusing.. what does this check do?

Reply:
I improved the name.